elements/urth-core-dataframe/urth-core-dataframe.html

```diff
@@ -339,6 +367,16 @@
         },
 
         /**
+         * Returns the type information of the serialized Dataframe
+         * @param df - the serialized DataFrame
+         * @return the types array
+         * @private
+         */
+        _columnTypes: function(df){
```

Comment:
Did you mean documentation on the wiki at https://github.com/jupyter-incubator/declarativewidgets/wiki/Using-DataFrames (I'll do that once this gets merged in) or something else?